src/prices/repository.js

```diff
@@ -86,13 +87,13 @@ async function createPrice(db, hutPrice) {
                 {
                     createdAt: now,
                     updatedAt: now,
-                    ...hutPrice
+                    ...priceData
                 }
             ]
         );
     } catch(error) {
         throw new Error("failed to insert a new price into 'prices' table");
     }
-
-    return;
+    
```

Comment:
Spaces on this line are odd

Reply:
Oops, just noticed that this is the prices repo that is removed again in next commit. But the feedback stands :smile: